### ISSUE TITLE
Plotting multiple indicators on the same "subplot" using "plotmaster"

### ISSUE THREAD

**Issue by purinda** [verybadsoldier/backtrader_plotting#30]
Library seem to have a bug related to plotting more than one indicator on a subplot. 

Following is a duplicate of the tradimo demo you have which I modified to add two SMA indicators with different periods, 20 and 5. I want these two SMAs to be plotted on the same subplot but the usual Backtrader way of sub-plotting doesn't seem to work here.

At the bottom I have attached the Backtrader matpotlib output vs backtrader_plotting library (Bokeh) output. 

```
import datetime
import backtrader as bt
from backtrader_plotting import Bokeh
from backtrader_plotting.schemes import Tradimo


class MyStrategy(bt.Strategy):
    def __init__(self):
        sma = bt.indicators.SimpleMovingAverage(period=20, subplot=True)
        sma2 = bt.indicators.SimpleMovingAverage(period=5, subplot=True, plotmaster=sma)

    def next(self):
        pos = len(self.data)
        if pos == 45 or pos == 145:
            self.buy(self.datas[0], size=None)

        if pos == 116 or pos == 215:
            self.sell(self.datas[0], size=None)


if __name__ == '__main__':
    cerebro = bt.Cerebro()

    cerebro.addstrategy(MyStrategy)

    data = bt.feeds.YahooFinanceCSVData(
        dataname="datas/orcl-1995-2014.txt",
        fromdate=datetime.datetime(2000, 1, 1),
        todate=datetime.datetime(2001, 2, 28),
        reverse=False,
    )
    cerebro.adddata(data)
    cerebro.addanalyzer(bt.analyzers.SharpeRatio)

    cerebro.run()

    b = Bokeh(filename='chart.html',style='bar', plot_mode="single", scheme=Tradimo())
    cerebro.plot(b)

    # Try out matplotlib output
    cerebro.plot()
```

Matpotlib (on the same subplot using `plotmaster`)
![Screen Shot 2019-04-26 at 5 25 41 pm](https://user-images.githubusercontent.com/3181510/56790991-5b967880-6849-11e9-8981-a91b27bb1549.png)

Bokeh html out (two subplots)
![Screen Shot 2019-04-26 at 5 26 06 pm](https://user-images.githubusercontent.com/3181510/56791004-62bd8680-6849-11e9-86a8-65a01a1b5c2d.png)

**Comment by purinda** [comment]
@verybadsoldier I had some time today to look into the project code. I managed to fix the immediate problem of incorrect resolution and PR is https://github.com/verybadsoldier/backtrader_plotting/pull/31

If looks okay, please merge and tag, keen to get the package updated. 

Cheers

**Comment by jesseliu0** [comment]
@purinda 
I installed your fix by pip on my Mac/Python 3.7.3 backtrader 1.9.74.123, and tried your sample code, only changing offline data to online data, but no avail. Screenshot and code attached below. Thanks.

![image](https://user-images.githubusercontent.com/4315932/62034418-d6519580-b1a2-11e9-8e7b-8c894bf181a2.png)

```python
import datetime
import backtrader as bt
from backtrader_plotting import Bokeh
from backtrader_plotting.schemes import Tradimo


class MyStrategy(bt.Strategy):
    def __init__(self):
        sma = bt.indicators.SimpleMovingAverage(period=20, subplot=True)
        sma2 = bt.indicators.SimpleMovingAverage(period=5, subplot=True, plotmaster=sma)

    def next(self):
        pos = len(self.data)
        if pos == 45 or pos == 145:
            self.buy(self.datas[0], size=None)

        if pos == 116 or pos == 215:
            self.sell(self.datas[0], size=None)


if __name__ == '__main__':
    cerebro = bt.Cerebro()

    cerebro.addstrategy(MyStrategy)

    data = bt.feeds.YahooFinanceData(
        dataname='orcl',
        fromdate=datetime.datetime(2000, 1, 1),
        todate=datetime.datetime(2001, 2, 28),
        reverse=False,
    )
    cerebro.adddata(data)
    cerebro.addanalyzer(bt.analyzers.SharpeRatio)
    cerebro.run()

    b = Bokeh(filename='chart.html',style='bar', plot_mode="single", scheme=Tradimo())
    cerebro.plot(b)
```

**Comment by verybadsoldier** [comment]
Fixed in latest `develop`. Sorry for the delay.
